README-maintainer

```diff
@@ -323,7 +323,7 @@ Template for release notes:
 ```
 This is qpdf version x.y.z. (Brief description)
 
-For a full list of changes from previous releases, please see the [release notes](http://qpdf.sourceforge.net/files/qpdf-manual.html#release-notes). See also [README-what-to-download](./README-what-to-download.md) for details on the available source and binary distributions.
+For a full list of changes from previous releases, please see the [release notes](http://qpdf.sourceforge.net/files/qpdf-manual.html#release-notes). See also [README-what-to-download](./README-what-to-download.md) for details of the available source and binary distributions.
```

Comment:
I would accept changing `on` to `about` but not `of`. Otherwise all the other corrections look good. If you're willing to help proofread the docs when I've had a change to make my own pass, I'd graciously accept and give you credit in the docs. I had another defunct open source project in 2007 (started as a work project but got permission to go open source with it), and a coworker painstakingly proofread the 200 page docs twice.

Reply:
I agree about reads better. Added two more changes.

Don't know about the painstaking part, but quite happy to give a thorough read.